docs/exploration-and-organization/collections.md

```diff
@@ -82,6 +82,18 @@ Metabase will create a [model](../data-modeling/models.md) that contains that CS
 
 Uploads will only be available if your admin has enabled uploads for your Metabase, and you're in a group with Unrestricted access to the schema used to store those uploads. See [Uploading data](../databases/uploads.md).
 
+## Appending to a model created by an upload
+
+You can upload additional CSV data to an existing model created by a previous CSV upload.
+
+![Append data to existing upload model](./images/append-data.png)
+
+The uploaded CSV must have the same column and ordering as the CSV that created the model. Metabase will look for a header row to check that the column names are the same. So if you split a large CSV into multiple CSVs, make sure to include header rows for all of the files.
```

Comment:
I'd dedicate this paragraph only to compatibility (and talk about large csv splits elsewhere, likely in the other section about splits)

I should mention: column name, column order, and column type

I think it is worth mentioning which coercions we support. I'm not sure which coercions we support now (if any). Something like: if I upload float data to integer column, the column should be converted to float.